pkg/metadata/metadata.go

```diff
@@ -25,6 +25,8 @@ const (
 	RebootAnnotation = "windowsmachineconfig.openshift.io/reboot-required"
 	// UpgradingLabel indicates the node's underlying instance is performing an upgrade
 	UpgradingLabel = "windowsmachineconfig.openshift.io/upgrading"
+	// TlsAnnotation indicates the node's TLS certs need update
+	TlsAnnotation = "windowsmachineconfig.openshift.io/tls-update"
```

Comment:
```suggestion
	TlsUpdateRequiredAnnotation = "windowsmachineconfig.openshift.io/tls-update-required"
```

so that is clear the node needs attention. Other suggestions:
-  `tls-mismatch`
-  `tls-failure`